tests/test_utils.py

```diff
@@ -44,3 +47,51 @@ def test_read_header(input_dir):
     assert (sr, frames, sampwidth, channels, size) == read_header(
         input_dir.joinpath("test.wav")
     )
+
+
+@pytest.fixture
+def correct_dataframe():
+    data = pd.DataFrame(
+        {
+            "dataset": ["dataset_test", "dataset_test", "dataset_test"],
+            "filename": ["file1.wav", "file2.wav", "file3.wav"],
+            "start_time": [0, 0, 5.9],
+            "end_time": [60, 60, 8.1],
+            "start_frequency": [0, 0, 18500.0],
+            "end_frequency": [96000, 96000, 53000.0],
+            "annotation": ["boat", "boat", "boat"],
+            "annotator": ["bbjuni", "bbjuni", "bbjuni"],
+            "start_datetime": [
+                pd.Timestamp("2020-05-29T11:30:00.000+00:00"),
+                pd.Timestamp("2020-05-29T11:31:00.000+00:00"),
+                pd.Timestamp("2020-05-29T11:31:00.000+00:00"),
+            ],
+            "end_datetime": [
+                pd.Timestamp("2020-05-29T11:31:00.000+00:00"),
+                pd.Timestamp("2020-05-29T11:32:00.000+00:00"),
+                pd.Timestamp("2020-05-29T11:32:00.000+00:00"),
+            ],
+            "is_box": [0, 0, 1],
+        }
+    )
+
+    return data.reset_index(drop=True)
+
+
+@pytest.mark.unit
+def test_aplose2raven(correct_dataframe):
+    df_raven = aplose2raven(df=correct_dataframe)
+
+    expected_dataframe = pd.DataFrame(
```

Comment:
Same here, maybe naming the variables `raven_dataframe` and `expected_raven_dataframe` would make the thing 100% explicit